edx_proctoring/static/proctoring/js/views/proctored_exam_view.js

```diff
@@ -43,6 +45,17 @@ var edx = edx || {};
             /* will call into the rendering */
             this.model.fetch();
         },
+        detectScroll: function(event) {
+            if ($(event.currentTarget).scrollTop() > this.timerBarTopPosition) {
+                $(".proctored_exam_status").addClass('fixed');
+                $(".wrapper-course-material").css('margin-top', '58px');
```

Comment:
can '58px' also be computed here rather than hard coded? It seems like it'd be very similar (maybe with an offset) to this.timerBarTopPosition...